src/main/java/com/rocketden/main/service/ProblemService.java

```diff
@@ -304,5 +306,10 @@ private void validateGsonParseable(String input, ProblemIOType type) {
         }
 
         return defaultCodeMap;
-    }   
+    }
+
+    public Boolean accessProblems(String password) {
```

Comment:
So I left this specifically un-tested because the test would either be checking that the problem access password equals the problem access password (because I cannot reveal the actual password itself in the codebase), or perhaps just testing that a random String will return false. Since the method is very straightforward and we don't want to reveal anything about the password, I leaned towards simply not testing this method.

I'll add a special comment to this method though, as you suggested.